### ISSUE TITLE
API GET config/routed_routed_networks

### ISSUE THREAD

**Issue by satkunas** [inverse-inc/packetfence#4182]
With the CRUD usage the `network` and `id` appear to be synonymous. 

This ambiguity should be removed and we should use either `id` or `network`, not both. To follow existing conventions `network` should become `id`.

**Comment by julsemaan** [comment]
@satkunas, fixed try again

**Comment by satkunas** [comment]
@julsemaan the collection is not populating the `id`

```json
{
  "items": [
    {
      "dhcp_default_lease_time": "3600",
      "dhcp_end": "2.0.0.1",
      "dhcp_max_lease_time": "7200",
      "dhcp_start": "2.0.0.0",
      "dhcpd": "disabled",
      "dns": "1.2.3.4",
      "fake_mac_enabled": 0,
      "gateway": "1.2.3.4",
      "id": null,
      "ip_assigned": null,
      "ip_reserved": null,
      "nat_enabled": 0,
      "netmask": "255.255.255.0",
      "next_hop": "2.3.4.5",
      "not_deletable": false,
      "portal_fqdn": null,
      "type": "vlan-isolation"
    }
  ],
  "nextCursor": 1,
  "prevCursor": 0,
  "status": 200
}
```

**Comment by julsemaan** [comment]
Should be fixed now in adf028363fe5118ae9c4ef4b5f009dc24794f24e
